pkg/uevent/utils.go

```diff
@@ -40,7 +40,7 @@ func getRootBlockPath(devName string) string {
 	}
 }
 
-func validateDevice(device *sys.Device, filteredDrives []*directcsi.DirectCSIDrive) bool {
+func ValidateDevice(device *sys.Device, filteredDrives []*directcsi.DirectCSIDrive) bool {
```

Comment:
we can export a single validate function which accepts a device and a drive

Like, 

```
func ValidateDriveStates(device *sys,Device, directCSIDrive *directcsi.DirectCSIDrive) bool {
        if directCSIDrive.Status.Path != device.DevPath() {
		return false
	}
	if directCSIDrive.Status.MajorNumber != uint32(device.Major) {
		return false
	}
	if directCSIDrive.Status.MinorNumber != uint32(device.Minor) {
		return false
	}
	if directCSIDrive.Status.Virtual != device.Virtual {
		return false
	}
	if directCSIDrive.Status.PartitionNum != device.Partition {
		return false
	}
	if directCSIDrive.Status.WWID != device.WWID {
		return false
	}
	if directCSIDrive.Status.ModelNumber != device.Model {
		return false
	}
	if directCSIDrive.Status.UeventSerial != device.UeventSerial {
		return false
	}
	if directCSIDrive.Status.Vendor != device.Vendor {
		return false
	}
	if directCSIDrive.Status.DMName != device.DMName {
		return false
	}
	if directCSIDrive.Status.DMUUID != device.DMUUID {
		return false
	}
	if directCSIDrive.Status.MDUUID != device.MDUUID {
		return false
	}
	if directCSIDrive.Status.PartTableUUID != device.PTUUID {
		return false
	}
	if directCSIDrive.Status.PartTableType != device.PTType {
		return false
	}
	if directCSIDrive.Status.PartitionUUID != device.PartUUID {
		return false
	}
	if directCSIDrive.Status.Filesystem != device.FSType {
		return false
	}
	if directCSIDrive.Status.UeventFSUUID != device.UeventFSUUID {
		return false
	}

	return true
}
```

and use this function here in L47. 

So, that we don't need to pass a list of drive as an input from the other places.